lib/nerves_hub_link/socket.ex

```diff
@@ -51,7 +51,8 @@ defmodule NervesHubLink.Socket do
     opts = [
       mint_opts: [protocols: [:http1], transport_opts: config.ssl],
       uri: config.socket[:url],
-      rejoin_after_msec: [@rejoin_after]
+      rejoin_after_msec: [@rejoin_after],
```

Comment:
Do you think we should apply the same jitter for rejoining channels as well?

Reply:
Why not? However the default values are different between the two

```
reconnect_after_msec: [
      ...
      default: [10, 50, 100, 150, 200, 250, 500, 1_000, 2_000, 5_000]
    ],
    rejoin_after_msec: [
      ...
      default: [100, 500, 1_000, 2_000, 5_000, 10_000]
    ],
```
I wondered how finely tuned these default values are, i.e. should they have different jittered backoffs or the same?

Reply:
Are channels vulnerable to thundering herds, i.e. does jittering matter?

Reply:
Unsure on both counts actually. I would imagine reconnect is more intensive than rejoining a channel which is essentially a simple socket message